AutoStructureFilesForDFplayer.spec

```diff
@@ -0,0 +1,33 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+block_cipher = None
+
+
+a = Analysis(['AutoStructureFilesForDFplayer.py'],
```

Comment:
My first project in Python. Is it common/usual to have the spec attached to the source code?

Reply:
its like a make file basically:
https://pyinstaller.readthedocs.io/en/stable/spec-files.html